core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -776,8 +776,8 @@ protected function generateButtons($arrRow, $strTable, $arrRootIds=array(), $bln
 			{
 				if (\is_array($v['label']))
 				{
-					$label = $v['label'][0];
-					$title = sprintf($v['label'][1], $id);
+					$label = $v['label'][0] ?? $k;
```

Comment:
For no behavior change it should be maybe:
```suggestion
					$label = $v['label'][0] ?? '';
```
But an operation without label makes no sense!